README.md

```diff
@@ -23,24 +23,44 @@ DOM at once in order to prevent jank. As an example, ReactJS is [adding the
 ability](https://reactjs.org/blog/2018/03/01/sneak-peek-beyond-react-16.html)
 to do async operations to help with this problem.
 
+Creating more complex applications might also mean having a **bigger amount of things in a web page**,
+making the rendering costs to have all of the content to actually be part of the DOM prohibitively expensive.
+
+This drives some amount of web authors to *virtualization* instead,
+where they actually **refrain from putting things in the DOM**,
+keeping content in memory in some non-DOM data structure,
+with only the visible portion converted into DOM nodes and inserted into the document,
+recycling them in and out as needed.
+
+However, this will **make the web app suffer** because a lot of things like find-in-page,
+accessibility, indexability, focus navigation, anchor links, etc.
+**depends on having things in the DOM**.
+
 ### Proposal
 
 This document proposes to augment user-agent APIs to help address the problem of
-slow rendering phase updates. Specifically, we propose a concept of display
-locking an element.
+**slow rendering phase updates**, and having some control on **when to pay rendering costs**
+Specifically, we propose a concept of *display locking* an element.
 
-If an element is locked for display, it means that any DOM updates to its
-subtree are not rendered immediately. Instead, when the lock is committed, the
-user-agent processed rendering updates co-operatively, yielding periodically to
-allow script or user interactions to happen.
+If an element is display locked, it means that
+**any DOM, style or layout updates to its subtree are not rendered immediately**.
+Instead, when allowed, the user agent will process rendering updates *co-operatively*,
+yielding periodically to allow script or user interactions to happen.
+The web author can also commit/unlock the lock,
+which triggers a synchronous update and render when necessary,
+causing the element to get updated and rendered in the next frame.
 
 The visual content of an element's subtree which is locked for display does not
-change. Specifically, if the element already existed in the DOM, then the
-content that was present at the time the lock was acquired remains. 
+change. When locked, if the element already existed in the DOM, then the
```

Comment:
So AFAIK "content that was present at the time the lock was acquired remains" is not true, at least in the current impl of Display Locking. I think a while ago it was determined that V1 of Display Locking won't retain the painted output for lock-after-append case. Is it still the case, and if so should we change this line too? @chrishtr @vmpstr

Reply:
Correct, we plan to skip this behavior in v1. Ok to remove.

Reply:
Changed the line to say it's like `visibility: hidden`